pkg/platform/runtime/setup/setup.go

```diff
@@ -421,7 +463,10 @@ func (s *Setup) setupArtifact(a artifact.ArtifactID, unsignedURI string, as Arti
 	unarchiver := as.Unarchiver()
 	var archivePath string
 	if s.target.InstallFromDir() != nil {
-		archivePath = unsignedURI
+		if !strings.HasPrefix(unsignedURI, "file://") {
+			return errs.New("Local artifacts must be in URI form (missing 'file://' prefix for %s)", unsignedURI)
+		}
+		archivePath = unsignedURI[len("file://"):]
```

Comment:
Please see https://github.com/ActiveState/cli/pull/1889#discussion_r897365543 and https://github.com/ActiveState/cli/pull/1889#discussion_r897366664